public/app/plugins/datasource/loki/LogContextProvider.ts

```diff
@@ -53,12 +53,17 @@ export class LogContextProvider {
     this.appliedContextFilters = [];
   }
 
-  private async getQueryAndRange(row: LogRowModel, options?: LogRowContextOptions, origQuery?: LokiQuery) {
+  private async getQueryAndRange(
+    row: LogRowModel,
+    options?: LogRowContextOptions,
+    origQuery?: LokiQuery,
+    forceApplyFilters?: boolean
+  ) {
     const direction = (options && options.direction) || LogRowContextQueryDirection.Backward;
     const limit = (options && options.limit) || this.datasource.maxLines;
     // This happens only on initial load, when user haven't applied any filters yet
     // We need to get the initial filters from the row labels
```

Comment:
We need to update a comment to include both scenarios:
- when `this.appliedContextFilters.length === 0` happens only on initial load, when user haven't applied any filters yet and in that case we need to get the initial filters from the row labels
- if `forceApplyFilters === true` we want to apply initial filters from the row labels (which is currently used for `cmd + click` on show context)